Publisher/nl/email-editor-tests.md

```diff
@@ -11,7 +11,9 @@ Bij elke controle hoort een score die beschrijft in hoeverre de uitkomst van die
 Om te controleren hoe je mailing op desktop, tablet en mobile eruit ziet, ga je naar _Voorvertoning_ in de toolbar. De voorvertoning wordt standaard gepersonaliseerd op de ingestelde standaardbestemming. Rechtsbovenin kun je aangeven welk soort device je als voorbeeld wilt gebruiken. Daarnaast is het mogelijk om zelf de afmetingen van de preview aan te passen.
```

Comment:
Rechtsbovenin kun je aangeven welk soort device je als voorbeeld wilt gebruiken => Kan dit veranderen en is het nodig om naar de exacte locatie te wijzen?